source/NVDAObjects/IAccessible/adobeAcrobat.py

```diff
@@ -119,61 +119,96 @@ def _isEqual(self, other):
 			return self.accID == other.accID
 		return super(AcrobatNode, self)._isEqual(other)
 
-	def _getNodeMathMl(self, node):
+	@staticmethod
+	def getMathMLAttributes(node, attrList: list) -> str:
```

Comment:
Node still needs a type hint

```suggestion
	def getMathMLAttributes(node: IPDDomElement, attrList: list) -> str:
```